daemon/cmd/endpoint.go

```diff
@@ -430,12 +430,13 @@ func (d *Daemon) createEndpoint(ctx context.Context, owner regeneration.Owner, e
 			}
 			addLabels.MergeLabels(identityLabels)
 			infoLabels.MergeLabels(info)
-			if _, ok := annotations[bandwidth.IngressBandwidth]; ok {
+			ingBandwidth := "kubernetes.io/ingress-bandwidth"
```

Comment:
How is this any better? I thought from your explanation that bandwidth.IngressBandwidth is unused and should be removed. Yet it is used in the code. So your change makes it worse.

Reply:
@borkmann thanks for the review. IMHO this is better b/c the annotation can no longer be imported by another pkg ([xref](https://go.dev/ref/spec#Exported_identifiers)). @pchaigno suggested in the associated Slack thread to remove the constant. Based on your feedback, I will update the PR to remove the associated log message.

Reply:
Did you consider simply not exporting the constant outside of the package where it's defined, eventually by moving it to another package?

Reply:
@pchaigno ^ is an option too. Does the team prefer the removal of the constant and associated log message or move the constant to the cmd pkg, make it internal, and keep the log message?